phpBB/styles/prosilver/theme/icons.css

```diff
@@ -11,20 +11,20 @@
  */
 .icon, .button .icon, blockquote cite:before, .uncited:before {
 	display: inline-block;
-  	font-weight: normal;
+  	font-weight: 900;
   	font-style: normal;
   	font-variant: normal;
-  	font-family: FontAwesome;
+  	font-family: 'Font Awesome 5 Free', 'Font Awesome 5 Brands';
```

Comment:
why are we adding brands hee if we are not using them? just for the user to take advantage? Not sure this even works like this. I put up a test for all the icons we use https://codepen.io/hanakin/pen/KKdNqow?editors=1100 the brand icons do not load either way we call them fa4 or fa5 class

Reply:
> just for the user to take advantage?

Sure. Isn't phpBB for users? ;) And yes, it works like that for me. And brand icons do load.

Reply:
So what do I do with that then?

Reply:
As I was i had a typo in the test